app/models/drawing.rb

```diff
@@ -4,4 +4,15 @@ class Drawing < ApplicationRecord
   belongs_to :planning_application
 
   has_one_attached :plan
+
+  enum archive_reason: { scale: 0, design: 1,
+                         dimensions: 2, other: 3 }
+
+  def is_archived?
+     archived_at == nil ? false : true
```

Comment:
How about `archived_at.present?`?

As I understand the method `is_archived?` is a helper method to check whether the `archived_at` timestamp is there

Reply:
Also how about rename it to `archived?`